akd_core/src/lib.rs

```diff
@@ -33,4 +33,4 @@ pub use types::*;
 
 /// The arity of the tree. Should EXACTLY match the ARITY within
 /// the AKD crate (i.e. akd::ARITY)
-pub const ARITY: usize = 2;
+pub const ARITY: u8 = 2;
```

Comment:
We should create a new issue tracking the removal of ARITY

Reply:
I made a comment about this in another comment, since we're removing a lot of the `ARITY` based loops, it's really only used now for array initialization which might be useful just as a name alias tbh. Just something to think about!

Reply:
Good point!